tool/webjar/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TEMP_DIR=$(mktemp -d)
+REPO_ROOT_DIR=${SCRIPT_DIR}/../..
+FLUTTER_BUILD_DIR=${REPO_ROOT_DIR}/build/web
+
+cd ${REPO_ROOT_DIR}
+flutter build web --base-href /registration-desk/
+
+cd ${SCRIPT_DIR}
+if [[ -d src ]]; then
+  rm -rf src/
+fi
+mkdir -p webjar/src/main/resources/META-INF/resources/registration-desk
+cp -r ${FLUTTER_BUILD_DIR}/* webjar/src/main/resources/META-INF/resources/registration-desk
+
+cp -r ${SCRIPT_DIR}/webjar/* ${TEMP_DIR}
+
+cd ${TEMP_DIR}
+git init
+git add .
+git commit -m "Add app compiled to HTML/JavaScript"
+
+pwd
+git remote add origin git@github.com:ksch-workflows/registration-desk.git
+
+VERSION="0.0.10"
+
+git tag $VERSION
```

Comment:
Is it possible to create patch releases for older versions with this approach?

Reply:
Should it remain possible to track from which code which release was created?

Reply:
Would it be better to create a separate "registration-desk-webjar" repository which contains the webjars?

Reply:
Maybe there could be two tags per version:

- 0.8.1-RELEASE
- 0.8.1